static/app/views/issueList/overview.tsx

```diff
@@ -1281,7 +1288,15 @@ class IssueListOverview extends Component<Props, State> {
 export default withRouteAnalytics(
   withApi(
     withPageFilters(
-      withSavedSearches(withOrganization(withIssueTags(withProfiler(IssueListOverview))))
+      withSavedSearches(
+        withOrganization(
+          withIssueTags(
+            withProfiler(
+              withExperiment(IssueListOverview, {experiment: 'PrioritySortExperiment'})
```

Comment:
@ceorourke I guess you don't actually need `withExperiment` the way you're reading the experiment value

Reply:
Although we will need `withExperiment` to actually log the experiment so i guess it's fine to leave it